patterns/posts-personal-blog.php

```diff
@@ -33,6 +24,15 @@
 		</div>
 		<!-- /wp:group -->
 	<!-- /wp:post-template -->
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
+		<!-- wp:query-no-results -->
+		<!-- wp:paragraph -->
+		<p><?php echo esc_html_x( 'Sorry, but nothing matched your search terms. Please try again with some different keywords.', 'Message explaining that there are no results returned from a search', 'twentytwentyfive' ); ?></p>
```

Comment:
I am not _searching for a term_ though 🤔

Reply:
Good point. I mostly went this route because the same results pattern is used on the search results page. That is _presumably_ also where you'll see this message, in practice, most of the time.

How about:

"Sorry, but nothing was found. Please try searching again."

Reply:
Or even, 

"Sorry, but nothing was found. Please try a search with different keywords."

Reply:
Yes that would work a bit better on all scenarios.

Reply:
This is just one template though. Some of the templates have a large amount of no results blocks:
https://github.com/WordPress/twentytwentyfive/pull/400